lib/feature/wallet/token_wallet_details/widgets/token_wallet_transactions/token_wallet_transactions_cubit.dart

```diff
@@ -45,11 +46,34 @@ class TokenWalletTransactionsCubit extends Cubit<TokenWalletTransactionsState> {
     );
   }
 
+  final _logger = Logger('TokenWalletTransactionsCubit');
+
   final Address owner;
   final Address rootTokenContract;
   final NekotonRepository nekotonRepository;
   final TokenWalletStorageService walletStorage;
 
+  /// If not null, then [_transactionsState] will check for coming new
+  /// transactions and there is no new, then [_canLoadMore] will be false.
+  String? _lastLtWhenPreloaded;
+  bool _isPreloadingTransactions = false;
+  bool _canLoadMore = true;
```

Comment:
Is it really ok that cubit contains part of it's state inside its own instance?

Reply:
-